file/validate.go

```diff
@@ -29,5 +31,70 @@ func validate(content []byte) error {
 		err := fmt.Errorf(desc.String())
 		errs.Errors = append(errs.Errors, err)
 	}
+
+	var kongdefaults *KongDefaults
+	if err := json.Unmarshal(content, kongdefaults); err != nil {
+		return fmt.Errorf("unmarshaling file into KongDefaults: %w", err)
+	}
+	if kongdefaults != nil {
+		if err := CheckDefaults(*kongdefaults); err != nil {
+			return fmt.Errorf("default values are not allowed for these fields: %w", err)
+		}
+	}
 	return errs
 }
+
+func Check(item string, t *string) string {
```

Comment:
Please lowercase this to not export this function.

Reply:
changed 67aa05d6a0a1841031b4910928070b31bfaf9be8